### PR TITLE
[doc]: update doc for python3.8 and pyenv for issue [#440]

### PR DESCRIPTION
**Pull request by AvengerMoJo**
Update the doc to include pyenv in setup script. 
Fixes: [#440]

Signed-off-by: Alex Lau (AvengerMoJo) <alau@suse.com>

### CODE REVIEW

File: doc/from-zero-to-hacking.md [hunk 1]
```diff
@@ -99,6 +99,15 @@ as well as bare-metal hardware environments.
 
    There will be a few warning from `npm` which you can safely ignore.
 
+   Aquarium require python3.8+ to run, if your host distribution does
+   not support python3.8+ offically yet, our setup script provide pyenv to
+   help you setup an alternative python environment. If pyenv is already
```
Comment (asettle) [inline doc/from-zero-to-hacking.md:104]:
```suggestion
   Aquarium requires python3.8+ to run successfully. If your host distribution does
   not support python3.8+ offically yet, our setup script provides pyenv to
   help you setup an alternative python environment. If pyenv already

File: doc/from-zero-to-hacking.md [hunk 2]
```diff
@@ -99,6 +99,15 @@ as well as bare-metal hardware environments.
...
+   exist in your $HOME/.pyenv/ , it will try to reuse your setup. Either
+   way you need to provide the python version you have or needed as follow:
+    `--pyenv-python 3.8.8`
```
Comment (asettle) [inline doc/from-zero-to-hacking.md:107]:
```suggestion
   exists in`$HOME/.pyenv/` , the script will try to reuse your setup. Regardless,
   you will need to provide the python version. For example:
    `--pyenv-python 3.8.8`

**Review by asettle — CHANGES_REQUESTED** [review]
Few nits :) looks good!

**Comment by kshtsk** [comment]
jenkins run leap

**Comment by kshtsk** [comment]
@asettle are your change requests addressed?

**Review by asettle — APPROVED** [review]

**Pull request merged**